microservices/first/utils/deployment_status.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum, auto
+
+class DeploymentStatus(Enum):
```

Comment:
Is this ever used? Struggling to find where

Reply:
I think this is for future-proofing when the Leader updates its status

Reply:
Makes sense, I'm cool with it staying in as long as we put some documentation on it

Reply:
What Owen said. This was more future proofing so we can add to it later and make sure our statuses are valid